### PR TITLE
Whiteboards with same name

### PR DESCRIPTION
**Pull request by Sumitwarrior7**
Creating multiple whiteboards with same name within a room is now restricted.

If a user attempts to change the board's name to one that already exists, an error message will be displayed

Closing Issue : #44 

Video Reference:

https://github.com/RocketChat/Apps.Whiteboard/assets/108853577/ac3fd3eb-2d6e-445c-b5c1-21cbba3afe96

### CODE REVIEW

**Comment by Sumitwarrior7** [comment]
now instead of sending message, notification is getting sent.

Video reference:

[Screencast from 16-12-23 11:41:51 PM IST.webm](https://github.com/RocketChat/Apps.Whiteboard/assets/108853577/d8896393-d6ca-4ed3-b2e8-6bb74794e5dc)

**Comment by Sumitwarrior7** [comment]
Here you can see in the video it is written "only you can see this message" in the error message

File: whiteboard/handlers/ExecuteViewSubmitHandler.ts [hunk 1]
```diff
@@ -93,7 +96,45 @@ export class ExecuteViewSubmitHandler {
...
+                                    .getMessageReader()
+                                    .getRoom(messageId);
+                                if (room) {
+                                    const newMessage = this.modify
+                                        .getCreator()
+                                        .startMessage()
+                                        .setSender(AppSender)
+                                        .setRoom(room)
+                                        .setText(
+                                            `"${newBoardname}" Whiteboard name is already taken. Try different name`
```
Comment (CulturalProfessor) [inline whiteboard/handlers/ExecuteViewSubmitHandler.ts:125]: check if bold boardname works and change message to ""${newBoardname}" Whiteboard name is already taken. Please try using a different name."

File: whiteboard/enum/uitlityEnum.ts [hunk 2]
```diff
@@ -39,4 +39,6 @@ export enum UtilityEnum {
     DELETE_MODAL_ID = "delete-modal-id",
     DELETE_ACTION_ID = "delete-action-id",
     DELETE_BLOCK_ID = "delete-block-id",
+    NAE_POPUP_MODAL_ID = "nae-popup-modal-id",
```
[the remark this answers is not in the document]
Comment (Sumitwarrior7) [reply whiteboard/enum/uitlityEnum.ts:42]: ok

File: whiteboard/handlers/ExecuteViewSubmitHandler.ts (hunk 1 above)
Comment (Sumitwarrior7) [reply whiteboard/handlers/ExecuteViewSubmitHandler.ts:125]: ok

Comment (Sumitwarrior7) [reply whiteboard/handlers/ExecuteViewSubmitHandler.ts:125]: Screenshot from 2023-12-22 23-37-07

is this looking good, i have made the boardname bold

**Comment by Sumitwarrior7** [comment]
I have updated the pr for the changes you have discussed above, so please check and tell me if any further changes need to be done @CulturalProfessor

**Comment by CulturalProfessor** [comment]
> I have updated the pr for the changes you have discussed above, so please check and tell me if any further changes need to be done @CulturalProfessor

keep commit messages in a format like `fix : something` or `feat : something `

**Pull request merged**